### PR TITLE
Fix 4085 cherry-pick

### PR DESCRIPTION
**Pull request by lordofhyphens**
Cherry pick of @bubnikv fix/workaround for #4085.

Thanks.

### CODE REVIEW

File: lib/Slic3r/GUI/3DScene.pm [hunk 1]
```diff
@@ -769,7 +778,24 @@ sub InitGL {
     # A handy trick -- have surface material mirror the color.
     glColorMaterial(GL_FRONT_AND_BACK, GL_AMBIENT_AND_DIFFUSE);
     glEnable(GL_COLOR_MATERIAL);
-    glEnable(GL_MULTISAMPLE);
+    glEnable(GL_MULTISAMPLE) if ($self->{can_multisample});
+
+}
+
+sub DestroyGL {
```
Comment (hyperair) [inline lib/Slic3r/GUI/3DScene.pm]: What's up with this extra sub?

File: lib/Slic3r/GUI/3DScene.pm [hunk 2]
```diff
@@ -744,7 +745,15 @@ sub InitGL {
     # Set antialiasing/multisampling
     glDisable(GL_LINE_SMOOTH);
     glDisable(GL_POLYGON_SMOOTH);
-    glEnable(GL_MULTISAMPLE);
+
+    # See "GL_MULTISAMPLE and GL_ARRAY_BUFFER_ARB messages on failed launch"
```
Comment (GilesBathgate) [inline lib/Slic3r/GUI/3DScene.pm]: I don't know what all this "junk" is for, my PR is cleaner and just contains the essentials : #4085

**Comment by GilesBathgate** [comment]
@lordofhyphens  I think this can also be closed now.

**Comment by lordofhyphens** [comment]
Duplicate of another PR #4230

**Pull request closed without merging**